crates/bevy_ecs/examples/events.rs

```diff
@@ -1,12 +1,12 @@
 //! In this example a system sends a custom event with a 50/50 chance during any frame.
 //! If an event was send, it will be printed by the console in a receiving system.
 
-use bevy_ecs::prelude::*;
+use bevy_ecs::{event::EventRegistry, prelude::*};
 
 fn main() {
     // Create a new empty world and add the event as a resource
     let mut world = World::new();
-    world.insert_resource(Events::<MyEvent>::default());
+    EventRegistry::register_event::<MyEvent>(&mut world);
```

Comment:
This could use a comment briefly explaining where the `EventRegistry` is stored.

Reply:
It appears to be a resource, but we should clarify that this inserts the resource if it does not exist.

Reply:
```suggestion
    // The event registry is stored as a resource, and allows us to quickly update all events at once.
    // This call adds both the registry resource and the events resource into the world.
    EventRegistry::register_event::<MyEvent>(&mut world);
```